endoreg_db/management/commands/validate_video_files.py

```
@@ -1,36 +1,39 @@
+            help="Check specific video ID",
         )
         parser.add_argument(
-            '--verbose',
-            action='store_true',
-            help='Enable verbose output',
+            "--verbose",
+            action="store_true",
+            help="Enable verbose output",
         )
 
     def handle(self, *args, **options):
```

Pylint (4.0.4)

[refactor] Too many branches (16/12)

(R0912)